theme/lumo/vaadin-context-menu-styles.html

```diff
@@ -86,6 +86,13 @@
 <dom-module id="lumo-context-menu-item" theme-for="vaadin-context-menu-item">
   <template>
     <style>
+      :host {
+        user-select: none;
+        -ms-user-select: none;
+        -moz-user-select: none;
```

Comment:
This can be removed too, see above comment.

Reply:
Removed.